src/main/java/com/student/user/app/controllers/item/AddStudentController.java

```diff
@@ -61,11 +63,13 @@ public class AddStudentController implements Initializable {
     @FXML
     private Button studentAddButton;
 
-    private StudentFacade studentFacade = new StudentFacadeImpl();
+
```

Comment:
Use the StudentInfoMgtApplication class instead. DAOs should not be used in UI codes

Reply:
copy po thank you po